### PR TITLE
Add README file

### PR DESCRIPTION
**Pull request by GenBrg**
Hello!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+hahaha
```
Comment (valentine710) [inline README.md:1]: Your readme file should be descriptive. Please revise it!

Comment (GenBrg) [reply README.md]: I've updated it!

File: README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Good README file
```
Comment (valentine710) [inline README.md:1]: This is better! Good job!

Comment (valentine710) [inline README.md:1]: Good work

**Pull request merged**